File: WebAPI/Models/TodoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebAPI.Models
{
    public class TodoModel
    { 
        public long Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string AuthorFullName { get; set; }
        public string ExecuterFullName { get; set; }
        public string ExecuterNickName { get; set; }
        public string Created { get; set; }
        public bool IsDone { get; set; }
        public int Priority { get; set; }
    }
}

File: ClientApp/src/app/models/todo.model.ts
export class TodoItem {
    public id:number;
    public title:string;
    public description:string;
    public authorFullName: string;
    public executerFullName: string;
    public executerNickName: string;
    public created: string; 
    public isDone:boolean; 
    public priority:number;

    constructor(
        _title:string, 
        _description:string,
        _author:string, 
        _exName:string, 
        _exNick:string, 
        _date:string, 
        _priority:number, 
        _isDone:boolean)
        {
            this.title = _title;
            this.description = _description;
            this.authorFullName = _author;
            this.executerFullName = _exName;
            this.executerNickName = _exNick;
            this.created = _date;
            this.priority = _priority;
            this.isDone = _isDone;
        }
}